changelogs/unreleased/upgrade-note-virtual-environments.yml

```diff
@@ -0,0 +1,11 @@
+---
+description: Update note regarding virtual environments.
+change-type: patch
+destination-branches: [iso4, iso3]
```

Comment:
Does this not apply to master as well (for OSS)?

Reply:
Yes. You are right. I forgot about OSS.